src/styles/Constants.ts

```diff
@@ -2,8 +2,8 @@ export enum Colours {
   Primary = '#05162F',
   Secondary = '#2E5584',
   SecondaryHover = '#284D7A',
-  Danger = '#AC3434',
-  DangerHover = '#9B2F2F',
+  Danger = '#B12323',
+  DangerHover = '#DD2B2B',
```

Comment:
what's prompting this change? do we have usages of either of these constants that are have different colours in the mocks?

Reply:
Charmaine made some changes to the colours, so I updated them accordingly.